.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+trim_trailing_whitespace = true
```

Comment:
looking at a lot of the code in the codebase, it appears this is the indentation style that should be respected